application/controllers/admin/themes.php

```diff
@@ -172,6 +172,16 @@ public function tmp($id)
      */
     public function upload()
     {
+        // Code for backward compatiblity with custom themes that are expecting to upload images to upload subaction. 
+        // That happens as their options.twig are outdated.
+        // Now that the upload subaction doesn't handle all uploads, weneed to dispatch to the proper.
+        $action = returnGlobal('action');
```

Comment:
`returnGlobal` **must** be deprecated … please …

Reply:
I understand, but keep in mind this is used 5 lines below in the same controller. Is this the right time to deprecate it?

Reply:
Right … (it was not deprecated before ??)